spec/rails_helper.rb

```diff
@@ -34,14 +34,20 @@
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
+  # loading seeds
+  config.before(:suite) do
+    Rails.application.load_seed 
+  end
```

Comment:
Maybe we do not want seed to be loaded in each specs.
I think is better to generate each rspec context in their own file. 
Each test will have specific needs.